packages/nextra-theme-docs/src/styles.css

```diff
@@ -558,58 +516,6 @@ input[type='search']::-webkit-search-results-decoration {
   @apply mr-1;
 }
 
-/* RTL */
-.nextra-container.rtl {
-  direction: rtl;
-  .nextra-sidebar ul ul {
-    @apply mr-5 ml-0;
-  }
-  .nextra-sidebar button,
-  .nextra-sidebar a {
-    @apply text-right;
-  }
-  article blockquote {
-    @apply border-r-2 border-l-0 pr-6 pl-0;
-  }
-}
-
-@screen md {
-  .nextra-container:not(.page) article .bleed.full {
-    margin-left: calc(-50vw + 50% + 8rem);
-    margin-right: calc(-50vw + 50% + 8rem);
-  }
-  .nextra-container.page article .bleed.full {
-    margin-left: calc(-50vw + 50%);
-    margin-right: calc(-50vw + 50%);
-  }
-}
-@screen xl {
-  .nextra-container:not(.rtl):not(.page) article .bleed.full {
-    margin-left: calc(50% - 50vw + 16rem);
-    margin-right: calc(50% - 50vw);
-  }
-  .nextra-container.rtl:not(.page) article .bleed.full {
-    margin-left: calc(50% - 50vw);
-    margin-right: calc(50% - 50vw + 16rem);
-  }
-  .nextra-container.page article .bleed.full {
-    margin-left: calc(50% - 50vw);
-    margin-right: calc(50% - 50vw);
-  }
-}
-.nextra-container.rtl {
-  code {
-    direction: ltr;
-  }
-  article ul,
-  article ol {
-    @apply ml-0 mr-6;
-  }
-  .nextra-sidebar {
-    @apply md:border-l;
-  }
```

Comment:
this was a mistake we don't need set border left because on ltr we don't have border right
<img width="441" alt="Screenshot 2022-08-13 at 00 42 03" src="https://user-images.githubusercontent.com/7361780/184454025-9bcf7de7-32d5-4d0c-89b0-60c01b56eb03.png">